### DISCUSSION TITLE
change Next Image default properties

### DISCUSSION THREAD

**Discussion by MhmdNdri** [vercel/next.js#27952] [Help]
I want to change lazy loading to eager for the entire project, not to change it one by one! Does anyone know how to do it?

**Comment by diegoaguilar** [comment]
Unless there's a very mixed up code, I'd do a regex based find and replace ;)

**Reply by MhmdNdri** [reply]
So there is no easy way? :-)

**Reply by diegoaguilar** [reply]
I think there's no way, right
